src/sentry/static/sentry/app/views/settings/components/forms/field/fieldQuestion.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+const FieldQuestion = styled('div')`
+  display: flex;
+  align-items: center;
+  margin-left: ${space(0.5)};
```

Comment:
Do you need these spacing / alignments styles?
Could you set a grid-gap on FieldLabel now that you've made it a grid?